tests/test_random_variables/test_categorical.py

```diff
@@ -0,0 +1,122 @@
+"""Tests for the categorical random variable."""
+
+
+import string
+
+import numpy as np
+import pytest
+
+from probnum import random_variables, utils
+
+NDIM = 5
+
+all_supports = pytest.mark.parametrize(
+    "support",
+    [
+        None,
+        np.arange(NDIM),
+        np.array(list(string.ascii_lowercase)[:NDIM]),
+        np.random.rand(NDIM, 3, 3),
+    ],
+)
+
+all_random_states = pytest.mark.parametrize(
+    "random_state",
+    [
+        None,
+        1,
+        np.random.default_rng(),
+    ],
+)
+
+
+@pytest.fixture
+def probabilities():
+    probabilities = np.random.rand(NDIM)
+    return probabilities / np.sum(probabilities)
+
+
+@pytest.fixture
+def categ(probabilities, support, random_state):
+    return random_variables.Categorical(
+        probabilities=probabilities, support=support, random_state=random_state
+    )
+
+
+@all_random_states
+@all_supports
+def test_probabilities(categ, probabilities):
+    assert categ.probabilities.shape == (NDIM,)
+    np.testing.assert_allclose(categ.probabilities, probabilities)
```

Comment:
Isn't that the same object reference? :D
We could test here whether or not the probabilities sum to one, but that's basically clear by the definition of the `probabilities()`-fixture, so... 🤷

Reply:
this test is mostly about being able to access `categ.probabilities` and that indeed the correct vector comes out. If one refactored `categ.probabilities -> categ.event_probabilities` we want a test to break. This is said test. 

Don't take the values of the vector too seriously though :)